challenges/models-info.py

```diff
@@ -0,0 +1,34 @@
+from django.db import models
+from django.utils import timezone
+
+class ContactRequests(models.Model):
+    name = models.CharField(max_length=100)
+    message = models.CharField(max_length=100)
+    date_posted = models.DateTimeField(default=timezone.now)
+
+def __str__(self):
+    return '%s %s' % (self.name, self.message)

```

Comment:
I think you intended this __str__ method to be a part of the ContactRequests class, but it's not because it's not indented.

Reply:
This string provides information regarding the object that I want visible when querying for it. For example when I'm in the Django shell and  I query within that table instead of returning object1, object2, .... I can actually see some of the fields associated with the object.